CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.19)
+project(ProGen)
+
+set(CMAKE_CXX_STANDARD 20)
+
+add_executable(ProGen src/main.cpp)
```

Comment:
Please add newline at EOF

Reply:
Haven't requested the review yet. Don't be so eager